scripts/bump_integration.sh

```diff
@@ -24,37 +31,41 @@ get_next_integration_version() {
     ((next_preview_number = preview_number + 1))
     next_preview_number_formatted=$(printf "%02d" "$next_preview_number")
     NEXT_INTEGRATION_VERSION="${current_version%-*}-preview${next_preview_number_formatted}"
-    echo "• Next integration version: $NEXT_INTEGRATION_VERSION"
+    echo "Next integration version: $NEXT_INTEGRATION_VERSION"
     export NEXT_INTEGRATION_VERSION
 }
 
 update_manifest_version_vars() {
     # cis_gcp
-    echo "• Update cloudshell_git_branch in manifest.yml"
+    echo "Update cloudshell_git_branch in manifest.yml"
     sed -i'' -E "s/cloudshell_git_branch=[0-9]+\.[0-9]+/cloudshell_git_branch=$MAJOR_MINOR_CLOUDBEAT/g" $MANIFEST_PATH
 
     # cis_aws + vuln_mgmt_aws
-    echo "• Update cloudformation-* in manifest.yml"
+    echo "Update cloudformation-* in manifest.yml"
     sed -i'' -E "s/cloudformation-cnvm-[0-9]+\.[0-9]+\.[0-9]+/cloudformation-cnvm-$CURRENT_CLOUDBEAT_VERSION/g" $MANIFEST_PATH
     sed -i'' -E "s/cloudformation-cspm-ACCOUNT_TYPE-[0-9]+\.[0-9]+\.[0-9]+/cloudformation-cspm-ACCOUNT_TYPE-$CURRENT_CLOUDBEAT_VERSION/g" $MANIFEST_PATH
+    sed -i'' -E "s/cloudformation-cspm-direct-access-key-ACCOUNT_TYPE-[0-9]+\.[0-9]+\.[0-9]+/cloudformation-cspm-ACCOUNT_TYPE-$CURRENT_CLOUDBEAT_VERSION/g" $MANIFEST_PATH
```

Comment:
this was added to update the [new credentials deployment variable](https://github.com/elastic/integrations/pull/9828/files#diff-72fc27a4cf2fb28244c8fa9ed27c00e43935777a4875c5d61d91372f0d483425R127)